### ISSUE TITLE
Max columns in high dpi devices

### ISSUE THREAD

**Issue by Buxuna** [SkyD666/AniVu#63]
I have a small tablet but the dpi is quite high (more than 1000) so I increased  the font size to make up for the dpi , but in anivu in the left screen the columns became two and for that the title length and description length I see compared to when the dpi is less than 844 is significantly lower ,if an ability to limit the Max columns (in my case I want max 1 ) in high dpi devices is added it will be very helpfull . 

Thank u very much for making such a great app

**Comment by Buxuna** [comment]
And in portrait mode it looks horrible in a my device .
![pic](https://github.com/SkyD666/AniVu/assets/171773426/263aa36e-037f-41ba-9dc4-23fc50e36da8)

**Comment by SkyD666** [comment]
Expected behavior, multiple columns will be displayed on wide screens

**Comment by Buxuna** [comment]
Yeah i understand that , but my device is not wide its about 10 inch and only the dpi is high , the two column doesnt look good for that reason also if a name is just a little long it doesnt fit . A optional max column limit would really be appreciated

**Comment by SkyD666** [comment]
Maybe I will add a configuration later to support modifying the minimum width limit

**Comment by cauxar** [comment]
> Yeah i understand that , but my device is not wide its about 10 inch and only the dpi is high , the two columns don't look good for that reason also if a name is just a little long it doesnt fit . A optional max column limit would really be appreciated 

Yeh bro the same with me but for me its my phone I increased my dpi for games and suddenly there were two columns I was worried why . It didnt look good and I just came to file an issue for that , god this will be an helpful feature

**Comment by Buxuna** [comment]
Thank you very much for implementing this, however I noticed that this doesnt apply for articles in search screen (article search screen not feed search page ) if it's implemented  it will be a great help because the issue mostly remains for me in search screen and I use search a lot , 

Thank you very much for the live search , its really responsive

**Comment by Buxuna** [comment]
https://m3.material.io/foundations/layout/canonical-layouts/list-detail#50fd162f-6a89-4af0-b820-da080b7e45b8 


Also can we have the max value of minimum width slider atleast 800 dp for article search item width  because it makes 2 columns in portraing and 3 columnsin landscape  in the search page

**Comment by SkyD666** [comment]
> Thank you very much for implementing this, however I noticed for the search screen it applies for both feed list and article list , while I'm grateful that it applies for search articles but the feeds names are short and they are taking up one whole row which is very un-elegant looking , if the issue is resolved and the min width is not applied for feed search it will be a great help.
> 
> Thank you very much for the live search , its really responsive ♥️

For the search screen, due to the code implementation, there is no special configuration for the feed item at this time.

**Comment by SkyD666** [comment]
For 800dp, see https://github.com/SkyD666/AniVu/commit/4299e88df2014471cc0b76714425181a23d55ad3

**Comment by Buxuna** [comment]
> For 800dp, see https://github.com/SkyD666/AniVu/commit/4299e88df2014471cc0b76714425181a23d55ad3

Damn 1000 thanks
